spring-integration-core/src/test/java/org/springframework/integration/config/InvalidPriorityChannelParserTests.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2002-2016 the original author or authors.
```

Comment:
I said you before: don't comment if you agree. 
Just fix and add simple commend `Pushed`.

Only different opinion makes sense. :smile: